### DISCUSSION TITLE
Is there a Linux Guide?

### DISCUSSION THREAD

**Discussion by ankomen2** [swar/Swar-Chia-Plot-Manager#865] [Q&A]
I can't seem to get it started with the instructions provided. Step by Step somewhere?

**Comment by wesray** [comment]
This video may help. https://www.youtube.com/watch?v=Wl6FVRAj86I

Where's the hang-up, i can about 20 mins I can answer a few questions

**Reply by ankomen2** [reply]
I will watch the video. Thanks. The problem was the location of the chia executable.

**Comment by stou80** [comment]
Hi

I'm also interested to have a step-by-step to install swar on linux. 

Thanks
